update/delta/delta_1.3.2_fix_valve_missing_constraints.sql

```diff
@@ -1,7 +0,0 @@
-
-
--- Missing valve constraints
-
--- ALTER TABLE qwat_od.valve ADD CONSTRAINT chk_valve_altitude_obj_ref CHECK (fk_object_reference IS NOT NULL OR altitude IS NULL );
--- ALTER TABLE qwat_od.valve ADD CONSTRAINT chk_valve_altitude_precisionalti CHECK (fk_precisionalti IS NOT NULL OR altitude IS NULL );
-SELECT 1;
```

Comment:
This was a rather strange delta file! Do you have explanations for this?

Reply:
That was in that PR  https://github.com/qwat/qwat-data-model/pull/219/files , but i don't remember why it is commented... I propose to leave the comment for the moment, and clean them once we'll be sure it's not an error.

Reply:
You are the one who commented it in https://github.com/qwat/qwat-data-model/commit/d3dbec4b6f39e7789947e40e1b649690d9959f61.

Reply:
Yes. Don't remember the reason.

Reply:
In that case I don't see the reason for keeping them :)

Reply:
Ok right. Deleted.